src/services/Viewer.js

```diff
@@ -90,7 +93,7 @@ export default class Viewer {
 	 * @param {string} path the path to open
 	 * @param {Object[]} [list] the list of files as objects (fileinfo) format
```

Comment:
Could you also update the jsdoc, it is outdated and doesn't reflect the Object params requirements (I forgot to update it before, but since we're at it :see_no_evil:)